src/main/docker/Dockerfile

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2015-2017 EntIT Software LLC, a Micro Focus company.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM cafapi/opensuse-jre8:1.2
+
+# Update the OS packages and install Python
+RUN zypper -n refresh && \
+    zypper -n update && \
+    zypper -n install python-devel python-pip python-setuptools libxml2-devel libxslt-devel zlib-devel python-numpy python-lxml python-scipy python-matplotlib
```

Comment:
Use the line continuation character so that this doesn't run over the guideline.